compiler-core/src/type_/tests.rs

```diff
@@ -199,7 +212,15 @@ pub fn stringify_tuple_strs(module: Vec<(&str, &str)>) -> Vec<(SmolStr, String)>
         .collect()
 }
 
-pub fn infer_module(src: &str, dep: Vec<(&str, &str)>) -> Vec<(SmolStr, String)> {
+/// A module loaded as a dependency in the tests.
+///
+/// In order the tuple elements indicate:
+/// 1. The package name.
+/// 2. The module name.
+/// 3. The module source.
+type DependencyModule<'a> = (&'a str, &'a str, &'a str);
```

Comment:
With a third tuple element being added I felt the inline type definition was starting to get a bit unclear.

I moved it out into a type alias to document what each element of the tuple represents.

I think a struct would probably be clearer, but it's a bit more complicated to refer to from within the macro (but I can go that way, if we think it's worth it).